examples/pytorch/seal/main.py

```diff
@@ -0,0 +1,175 @@
+import dgl
```

Comment:
Usually we will follow some orders on import:
1.Original python packages, like os, sys;
2.Commonly used packages, like numpy, scipy, tqdm, etc;
3.The specific packages you are using, like dgl;
4.Code in other files in this example.